common/src/main/java/com/github/khanshoaib3/minecraft_access/utils/condition/Keystroke.java

```diff
@@ -131,4 +147,12 @@ public boolean aboutToHappen(Keystroke keystroke) {
             return this.preCondition.apply(keystroke);
         }
     }
+
+    /**
+     * Let this method handle state updates,
+     * no need to manually call the update method in every feature.
+     */
+    public static void updateAllExistingKeyStrokesStates() {
```

Comment:
Something like `updateKeystrokes` would be more clear.
```suggestion
    public static void updateKeystrokes() {
```